driver/opentelekomcloud.go

```diff
@@ -49,6 +50,19 @@ const (
 	defaultAuthURL       = "https://iam.eu-de.otc.t-systems.com/v3"
 	defaultVpcName       = "vpc-docker-machine"
 	defaultSubnetName    = "subnet-docker-machine"
+	k8sGroupName         = "sg-k8s"
+)
+
+var (
+	// https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#check-required-ports
+	k8sPorts = []services.PortRange{
+		// control-plane node(s)
+		{From: 6443},
+		{From: 2379, To: 2380},
```

Comment:
2379 is etcd port

Reply:
Comment notice worker and control-nodes "rule sets"